src/compas/geometry/__init__.py

```diff
@@ -1250,6 +1284,13 @@
 
     'Collection',
     'PointCollection',
+
+    'Curve',
+    'BezierCurve',
+    'NurbsCurve',
```

Comment:
Does this and the surface need to be bumped down into the conditional?

Reply:
not anymore